src/UserGuide/V1.2.x/User-Manual/Tiered-Storage_timecho.md

```diff
@@ -19,6 +19,78 @@
+3. configure the minimum remaining storage space ratio for each tier so that when the storage space of the tier triggers the threshold, the data of the tier will be automatically migrated to the next tier (optional).
+
+The specific parameter definitions and their descriptions are as follows.
+
+| Configuration                                    | Default                    | Description                                                          | Constraint                                                          |
+| ---------------------------------------- | ------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| dn_data_dirs                             | None                        | specify different storage directories and divide the storage directories into tiers             | Each level of storage uses a semicolon to separate, and commas to separate within a single level; cloud configuration can only be used as the last level of storage and the first level can't be used as cloud storage; a cloud object at most; the remote storage directory is denoted by OBJECT_STORAGE |
+| default_ttl_in_ms                        | None                        | Define the scope of data for which each tier is responsible, expressed through a TTL                    | Each level of storage is separated by a semicolon; the number of levels should match the number of levels defined by dn_data_dirs |
+| dn_default_space_move_thresholds         | 0.15                     | Define the minimum remaining space ratio for each tier data catalogue; when the remaining space is less than this ratio, the data will be automatically migrated to the next tier; when the remaining storage space of the last tier falls below this threshold, the system will be set to READ_ONLY | Each level of storage is separated by a semicolon; the number of levels should match the number of levels defined by dn_data_dirs |
+| object_storage_type                      | AWS_S3                   | Cloud Storage Type                                                 | IoTDB currently only supports AWS S3 as a remote storage type, and this parameter can't be modified   |
+| object_storage_bucket                    | None                        | Name of cloud storage bucket                                       | Bucket definition in AWS S3; no need to configure if remote storage is not used        |
+| object_storage_endpoiont                 |                          | endpoint of cloud storage                                          | endpoint of AWS S3；If remote storage is not used, no configuration required             |
+| object_storage_access_key                |                          | Authentication information stored in the cloud: key                                       | AWS S3 的 credential key；If remote storage is not used, no configuration required       |
```

Comment:
| object_storage_access_key                |                          | Authentication information stored in the cloud: key                                       | AWS S3 credential key；If remote storage is not used, no configuration required       |